pages/nft/url/index.vue

```diff
@@ -29,13 +29,52 @@
         <img v-if="avatar" :src="avatar">
         <Label :text="$t('NFTPortal.errorMessage.notIscnOwner', { ownerWallet })" />
       </div>
-      <div :class="[
-        'flex',
-        'flex-col',
-        'justify-center',
-        'items-center',
-        'w-full',
-      ]">
+
+      <div v-if="!isReady" class="flex justify-center items-center py-[64px]">
+        <ProgressIndicator />
+      </div>
+      <!-- Not Whitelisted -->
+      <div
+        v-else-if="!isAllowed"
+        class="flex flex-col my-[64px]"
+      >
+        <div
+          :class="[
+            'flex',
+            'items-start',
+            'justify-start',
+            'gap-[8px]',
+            'px-[12px]',
+            'py-[8px]',
+            'bg-[#FCF1DC]',
+            'border-[1px]',
+            'border-[#F3C267]',
+            'rounded-[4px]',
```

Comment:
Visually more balance since app.like.co has many rounded corners
```suggestion
            'rounded-[12px]',
```